.github/workflows/main.yml

```diff
@@ -10,20 +10,17 @@ on:
       - '*/*'       # matches every branch containing a single '/'
       - '**'        # matches every branch
   workflow_dispatch:
-    inputs:
-      comment:
-        description: 'Add comment for manual workflow execution.'
-        required: false
-        default: 'Manual execution of Github workflow.'
 
 jobs:
   pre-release:
     name: "Pre Release"
     runs-on: "ubuntu-latest"
     
     steps:
-      - uses: VatsalJagani/splunk-app-action@v1
+      - uses: VatsalJagani/splunk-app-action@v2
         with:
-          app_build_name: "splunk_app_3cx"
+          app_dir: "."
```

Comment:
Is it fine? or should we make file structure as other addons for consistency.

Reply:
Yes, this is fine. We have 2-3 add-ons which has root directory as app root.